src/FixedDataTable.js

```diff
@@ -727,21 +732,25 @@ var FixedDataTable = React.createClass({
     reorderingData.columnBefore = undefined;
     reorderingData.columnAfter = undefined;
 
+    var isFixedColumn = this.state.columnReorderingData.isFixed;
     var scrollX = this.state.scrollX;
-    //Relative dragX position on scroll
-    var dragX = reorderingData.originalLeft - reorderingData.scrollStart + reorderingData.dragDistance;
 
-    var fixedColumnsWidth = this.state.bodyFixedColumns.reduce((sum, column) => sum + column.props.width, 0);
-    var relativeWidth = this.props.width - fixedColumnsWidth;
+    if (!isFixedColumn) {
```

Comment:
interesting, so this means we won't scroll through fixed columns but I guess there's no use case for that.